lua/nvim-treesitter/parsers.lua

```diff
@@ -326,6 +326,15 @@ list.elixir = {
   maintainers = { "@jonatanklosko" },
 }
 
+list.gleam = {
+  install_info = {
+    url = "https://github.com/J3RN/tree-sitter-gleam",
+    files = { "src/parser.c" },
+    branch = "main",
+  },
+  maintainers = { "@J3RN" },
```

Comment:
You would be the maintainer for nvim-treesitter for this parser.

Reply:
For the maintainers section, is it intended to be the maintainer of the queries and tests in nvim-treesitter, or the maintainer of the parser itself? For example, I am the maintainer for tree-sitter-heex and the repository lives under my namespace, but I did not write tree-sitter-gleam.